server/app/views/applicant/ProgramCardsSectionFragment.html

```diff
@@ -26,6 +26,12 @@
 >
   <div class="usa-card__container">
     <div class="usa-card__header">
+      <img
+        th:if="${card.imageSourceUrl().isPresent() and not #strings.isEmpty(card.imageSourceUrl().get())}"
```

Comment:
(non-blocking) Is strings.isEmpty still needed?

Reply:
Yes, just tested it